README.md

```diff
@@ -24,11 +22,23 @@ npm i && code . && npm run dev
 
 ### The Problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Describe how you approached to problem: I first went through all of the instructions and hints within the code, making sure I understood each step, then I slowly line by line copied the helper code into the custom hook, add code along the way. My main problem was with Hint 9-11, creating a anchor to then save the download using the filname the user inputs with a prompt. I got this to work only by having new text appear on the page so the user had to first click "download" then the new link appeared and they would need to make a second click to "download" the oversion they have named. I was unsure how to dynamically create the <a> in the correct spot (I wanted it to encase the button).
+
+To solve this problem I changed my approach and focused on the donwload button. I changed the formattedFileName into a useState variable and then my downloadURL function was able to see it (I tried to pass it as props but this didnt work, then I remembered state!). It works this way.
+
+I was happy with my buttons, I used useState to hide and display all my buttons when they were relevant for the user.
+The UI begins with ONLY the Generation button and once generated the Download and Try Again buttons appear and the Generate and input field are hidden.
+
+
+What tools and techniques you used to solve it: I uses UseState to handle state manamgement, to handle passing the formatted file name to the download button, and to hide/display the buttons and input.
+
+What technologies did you use?: HTML5, CSS3, React, useState, custom Hooks, with Editor: VS Code
+
+If you had more time, what would be next?: I would do more styling, though I think I have found a font I really like and might use again. I would also explore the QR Library.
 
```

Comment:
Your readme is well written! I am always thinking about what to write and how to write, and I should look at your approach.

Reply:
And I also think the fond you chose seems fits nicely with other parts.